styles.css

```diff
@@ -1063,6 +1149,54 @@ textarea:focus {
   .heading {
     margin-top: 120px;
   }
+
+  .modal-container {
+    background: #fff;
+    border: 1px solid #ebecf0;
+    box-shadow: 48px 48px rgba(37, 47, 137, 0.08);
+    height: 794px;
+  }
+
+  /* .popup-content {
+   display: flex;
+   flex-direction: row;
+   margin-top: 12px;
+   gap: 12px;
+  } */
```

Comment:
- [ ] [OPTIONAL]  I would like to recommend you remove all the commented code from your codebase in order to keep your code clean :smile: